flow360/component/flow360_params/flow360_params.py

```diff
@@ -1529,7 +1699,7 @@ def to_flow360_json(self) -> dict:
         """
 
         solver_params = self.to_solver()
-        solver_params_json = solver_params.json(encoder=flow360_json_encoder)
+        solver_params_json = solver_params.json(exclude_unset=True, encoder=flow360_json_encoder)
```

Comment:
exclude_unset should not be changed, we need it for default settings used on webUI. What was the reason to add it here?

Reply:
Deleted. This was a residue from previous tries.